Source/WebKit/UIProcess/API/Cocoa/WKWebViewConfiguration.mm

```diff
@@ -292,6 +292,16 @@ - (instancetype)init
     return self;
 }
 
+- (void)setAllowsInlinePredictions:(BOOL)enabled
```

Comment:
264903@main (same change as the regression from the radar) caused this compiler warning. I just moved it out of the Private category to avoid the warning.